client/app/views/create-manual-order.html

```diff
@@ -39,12 +53,12 @@
                         <select id="selectStore" name="selectStore"
                                 class="form-control"
                                 ng-model="orderController.selectedStore"
-                                ng-options="store.name for store in orderController.stores"
+                                ng-options="store.name for store in orderController.stores | filter: !!store.isWarehouse"
```

Comment:
Is `!!store.isWarehouse` significantly different from `store.isWarehouse`?

Reply:
Yeah, I am just taking care of 'undefined' values